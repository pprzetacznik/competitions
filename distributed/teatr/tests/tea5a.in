100000000 0 -1 -1 1000000 5634398290718107035
