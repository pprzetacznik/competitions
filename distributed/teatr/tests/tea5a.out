2499996740994940
